robottelo/vm.py

```diff
@@ -380,6 +390,34 @@ def install_katello_ca(self):
             raise VirtualMachineError(
                 'Failed to download and install the katello-ca rpm')
 
+    def install_capsule_katello_ca(self, capsule=None):
+        """Downloads and installs katello-ca rpm on the virtual machine.
+        Uses common helper `install_capsule_katello_ca(self, capsule=None)`
```

Comment:
I think this sentence (that starts from ```Uses``` is no more valid now as we merged two functions into one now)